### PR TITLE
🐛 디테일 메타태그 url 수정

### PR DESCRIPTION
**Pull request by kykim00**
## 세부 사항
- 디테일 메타태그 url 수정

## 기타 질문 및 특이 사항
- meta tag 동적 세팅을 위해서 SSR을 구현했습니다.
- next-seo를 추가해서, 풀받으시면 yarn 하셔야합니다.
## 체크 리스트 (아래 사항들이 전부 체크되어야만 merge)
- [x]  필요한 test들을 완료하였고 기능이 제대로 실행되는지 확인 하였습니다.
- [x]  코드 스타일 가이드에 맞추어 코드를 작성 하였습니다.
- [x]  제가 의도한 파일들과 수정 사항들만 커밋이 된 것을 확인 하였습니다.
- [x]  본 수정 사항들을 팀원들과 사전에 상의하였고 팀원들 모두 해당 PR에 대하여 알고 있습니다.

### CODE REVIEW

**Comment by Junghoon-P** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>59edaff</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://space-dev-club-front-hgzxn3t71-junghoon-p.vercel.app'>https://space-dev-club-front-hgzxn3t71-junghoon-p.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/junghoon-p/space-dev-club-front/2T8u5PDnSf7CEmqN4zumeJvDbUKi'>https://vercel.com/junghoon-p/space-dev-club-front/2T8u5PDnSf7CEmqN4zumeJvDbUKi</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/SPACE-DEV-CLUB/SPACE_DEV_CLUB_FRONT/actions/runs/1971986227)

**Comment by Junghoon-P** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>b47e5bd</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://space-dev-club-front-rensq3ys5-junghoon-p.vercel.app'>https://space-dev-club-front-rensq3ys5-junghoon-p.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/junghoon-p/space-dev-club-front/DFoYXrHsfhaixJgVDjZ21hh8A5Ps'>https://vercel.com/junghoon-p/space-dev-club-front/DFoYXrHsfhaixJgVDjZ21hh8A5Ps</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/SPACE-DEV-CLUB/SPACE_DEV_CLUB_FRONT/actions/runs/1972017444)

**Comment by Junghoon-P** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>5b88ec8</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://space-dev-club-front-2i2lagfl5-junghoon-p.vercel.app'>https://space-dev-club-front-2i2lagfl5-junghoon-p.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/junghoon-p/space-dev-club-front/2uN7p55Q7mTFqCyvm8CLE6DWmfv1'>https://vercel.com/junghoon-p/space-dev-club-front/2uN7p55Q7mTFqCyvm8CLE6DWmfv1</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/SPACE-DEV-CLUB/SPACE_DEV_CLUB_FRONT/actions/runs/1975888856)

File: src/pages/[id]/[details]/index.tsx [hunk 1]
```diff
@@ -133,57 +135,50 @@ const DetailsIndexPage: NextPage = () => {
...
-        <meta name="url" property="og:url" content={postObj.url} />
-        <meta name="type" property="og:type" content="article" />
-        <meta name="title" property="og:title" content={postObj.title} />
-        <meta
-          name="description"
-          property="og:description"
-          content={postObj.contents}
-        />
+        <title>spacelog</title>
+        <meta property="og:title" content="sapcelog" />
```
Comment (deli-ght) [inline src/pages/[id]/[details]/index.tsx:174]: space 오타있어요

**Comment by Junghoon-P** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>7972eb7</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://space-dev-club-front-ley3mdu4u-junghoon-p.vercel.app'>https://space-dev-club-front-ley3mdu4u-junghoon-p.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/junghoon-p/space-dev-club-front/5h2vWymatF48BjwNLed7oGqZJ6Z6'>https://vercel.com/junghoon-p/space-dev-club-front/5h2vWymatF48BjwNLed7oGqZJ6Z6</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/SPACE-DEV-CLUB/SPACE_DEV_CLUB_FRONT/actions/runs/1975960083)

**Comment by Junghoon-P** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>c6ebe44</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://space-dev-club-front-38tocezbt-junghoon-p.vercel.app'>https://space-dev-club-front-38tocezbt-junghoon-p.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/junghoon-p/space-dev-club-front/CNioHTanrxW4Nh4TC4na8sbmr49P'>https://vercel.com/junghoon-p/space-dev-club-front/CNioHTanrxW4Nh4TC4na8sbmr49P</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/SPACE-DEV-CLUB/SPACE_DEV_CLUB_FRONT/actions/runs/1976013132)

**Comment by Junghoon-P** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>7b05212</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://space-dev-club-front-k1wu9aq35-junghoon-p.vercel.app'>https://space-dev-club-front-k1wu9aq35-junghoon-p.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/junghoon-p/space-dev-club-front/BLbhsWJw7gQhbraziz1kgKB1p5PC'>https://vercel.com/junghoon-p/space-dev-club-front/BLbhsWJw7gQhbraziz1kgKB1p5PC</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/SPACE-DEV-CLUB/SPACE_DEV_CLUB_FRONT/actions/runs/1976236616)

**Comment by Junghoon-P** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>438a8e6</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://space-dev-club-front-9qr1m9enp-junghoon-p.vercel.app'>https://space-dev-club-front-9qr1m9enp-junghoon-p.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/junghoon-p/space-dev-club-front/Gk6AwY2cqPZtmUvso2CpYygpw63U'>https://vercel.com/junghoon-p/space-dev-club-front/Gk6AwY2cqPZtmUvso2CpYygpw63U</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/SPACE-DEV-CLUB/SPACE_DEV_CLUB_FRONT/actions/runs/1980323949)

**Comment by Junghoon-P** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>91f1245</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://space-dev-club-front-p0voursdw-junghoon-p.vercel.app'>https://space-dev-club-front-p0voursdw-junghoon-p.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/junghoon-p/space-dev-club-front/CDGHDHZHCppPkiLJnMdLPmaqMjza'>https://vercel.com/junghoon-p/space-dev-club-front/CDGHDHZHCppPkiLJnMdLPmaqMjza</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/SPACE-DEV-CLUB/SPACE_DEV_CLUB_FRONT/actions/runs/1980400218)

**Comment by Junghoon-P** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>f5906b3</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://space-dev-club-front-bxt2ag3z6-junghoon-p.vercel.app'>https://space-dev-club-front-bxt2ag3z6-junghoon-p.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/junghoon-p/space-dev-club-front/BJMh4gVBEnpx5NuLxLxAoQarbZE6'>https://vercel.com/junghoon-p/space-dev-club-front/BJMh4gVBEnpx5NuLxLxAoQarbZE6</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/SPACE-DEV-CLUB/SPACE_DEV_CLUB_FRONT/actions/runs/1980447442)

**Comment by Junghoon-P** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>b9b626a</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://space-dev-club-front-mn4jccijw-junghoon-p.vercel.app'>https://space-dev-club-front-mn4jccijw-junghoon-p.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/junghoon-p/space-dev-club-front/DYdZ5ssp3ZPZLwkRZBqfWsAempep'>https://vercel.com/junghoon-p/space-dev-club-front/DYdZ5ssp3ZPZLwkRZBqfWsAempep</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/SPACE-DEV-CLUB/SPACE_DEV_CLUB_FRONT/actions/runs/1980509175)

**Comment by Junghoon-P** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>25799d9</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://space-dev-club-front-ckp89aigk-junghoon-p.vercel.app'>https://space-dev-club-front-ckp89aigk-junghoon-p.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/junghoon-p/space-dev-club-front/7w8kPj7heUYMGuUoyFjBa7UF6vNk'>https://vercel.com/junghoon-p/space-dev-club-front/7w8kPj7heUYMGuUoyFjBa7UF6vNk</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/SPACE-DEV-CLUB/SPACE_DEV_CLUB_FRONT/actions/runs/1980541788)

**Comment by Junghoon-P** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>a04c04a</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://space-dev-club-front-a4nia1e86-junghoon-p.vercel.app'>https://space-dev-club-front-a4nia1e86-junghoon-p.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/junghoon-p/space-dev-club-front/D3xJgnrocroy2QMEwz2emrhc3i6b'>https://vercel.com/junghoon-p/space-dev-club-front/D3xJgnrocroy2QMEwz2emrhc3i6b</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/SPACE-DEV-CLUB/SPACE_DEV_CLUB_FRONT/actions/runs/1987440553)

File: src/pages/[id]/[details]/index.tsx (hunk 1 above)
Comment (kykim00) [reply src/pages/[id]/[details]/index.tsx]: 수정했습니다.

**Comment by Junghoon-P** [comment]
수고하셨습니다.

**Pull request merged**